contentwarehouse/snippets/update_document_schema_sample_test.py

```diff
@@ -0,0 +1,40 @@
+# # Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import os
+
+from contentwarehouse.snippets import test_utilities
+from contentwarehouse.snippets import update_document_schema_sample
+import pytest
+
+project_id = os.environ["GOOGLE_CLOUD_PROJECT"]
+location = "us"
+document_schema_name = "YOUR_SCHEMA_NAME"
+
+
+def test_quickstart(capsys: pytest.CaptureFixture) -> None:
```

Comment:
change the name of this function
```suggestion
def test_update_document_schema_sample(capsys: pytest.CaptureFixture) -> None:
```